tests/docker_test.sh

```
@@ -0,0 +1,96 @@
+#!/bin/bash
+# Docker Image Test Script
+# Validates the Docker image across all supported architectures
+
+set -e
+
+IMAGE_NAME="${IMAGE_NAME:-ghcr.io/$(echo $GITHUB_REPOSITORY | tr '[:upper:]' '[:lower:]')}"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)